linux_os/guide/system/software/sudo/sudo_require_reauthentication/rule.yml

```diff
@@ -33,6 +33,7 @@ identifiers:
 
 references:
     cis@rhel8: 5.3.6
+    cis@rhel9: 5.3.6
```

Comment:
```suggestion
    cis@rhel9: 5.3.5,5.3.6
```

Based on the control file it should be this.

Reply:
Correct. Thanks